sdcm/tester.py

```diff
@@ -291,6 +291,20 @@ class ClusterInformation(NamedTuple):
     schema_versions: List[SchemaVersion]
 
 
+@dataclass
+class TabletsConfiguration:
```

Comment:
nice one,
consider moving it to `replication_strategy_utils.py`

Reply:
Will move, thanks